frontend/components/site/sitePublishedBranchesTable.js

```diff
@@ -1,26 +1,29 @@
 import React from "react";
 import { Link } from 'react-router';
 import publishedBranchActions from "../../actions/publishedBranchActions";
+import LoadingIndicator from "../loadingIndicator"
 
 class SitePublishedBranchesTable extends React.Component {
   componentDidMount() {
     publishedBranchActions.fetchPublishedBranches({ id: this.props.params.id })
   }
 
   publishedBranches() {
-    if (!this.props.publishedBranches) {
+    if (this.props.publishedBranches.isLoading || !this.props.publishedBranches.data) {
       return []
+    } else {
+      return this.props.publishedBranches.data
     }
-    return this.props.publishedBranches.filter(branch => {
-      return branch.site.id === parseInt(this.props.params.id)
-    })
   }
 
   render() {
-    if (this.publishedBranches().length > 0) {
-      return this.renderBuildLogsTable()
```

Comment:
This should not be named `renderBuildLogsTable`. Looks like an old copy/paste issue